services/comprehension/feedback-api-main/src/endpoint/endpoint.go

```diff
@@ -58,20 +58,31 @@ func GetOpinionDomain() (string) {
 	return opinion_domain
 }
 
+func GetGrammarDomain() (string) {
+	var maybe_domain = os.Getenv("grammar_domain")
+
+	var grammar_domain = "https://www.quill.org"
+
+	if len(maybe_domain) > 0 {
+		grammar_domain = maybe_domain
+	}
+	return grammar_domain
+}
+
```

Comment:
Why not re-use the pattern we have for other Evidence-engine-based APIs.  Like `automl_api` just uses `lms_domain`.